docs/concepts.md

```diff
@@ -48,7 +48,20 @@ A "region" is an entire interval of time (and not just a single point in time) t
 
 An entire recording has way too many objects that go in and out of existence and carry too many details for us to be able to hold all of that information in the memory at once. And that's why every recording is split into "regions" - smaller chunks of data that can be transmitted over network and stored in memory on the client, and further data derived from and interacted with. "Regions" are not something that user ever particularly cares about (beside being forced to wait for them to be loaded) and that they interact with, and instead, the user interacts with "pauses". A "pause" is that single moment in time for which the user can inspect the state of the application.
 
-A single `Pause` exists for each JS instruction. (TODO explain this and how it's related to debug stepping and `ExecutionPoint`s)
+A single `Pause` exists for each JS instruction. 
+
+#### How `Pause` is related to debug stepping and `ExecutionPoint`s
+
+In the Replay DevTools, a `Pause` represents the state of the JavaScript execution at a specific `ExecutionPoint`.Each `ExecutionPoint` corresponds to a specific line of code in the JavaScript program.
```

Comment:
> Each `ExecutionPoint` corresponds to a specific line of code in the JavaScript program.

Missing space between ".Each"

I think saying "a specific line" is maybe too broad/vague.

Execution points correspond to both a line and a column. (Lines can have multiple breakable columns.)

It also corresponds to a specific instance that code was executed within a recording. (A single location might have multiple "hits" in a recording.)